hedera-node/src/main/java/com/hedera/services/txns/token/TokenUpdateTransitionLogic.java

```diff
@@ -21,6 +21,7 @@
  */
 
 import com.hedera.services.context.TransactionContext;
+import com.hedera.services.exceptions.InvalidTransactionException;
```

Comment:
In the next week or so LimeChain will open a PR that completely refactors this class; I think we should revert these changes also to avoid needless merge conflicts.

Reply:
Absolutely. Done the reverting.